packages/backend-defaults/src/entrypoints/rootLifecycle/rootLifecycleServiceFactory.test.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { getVoidLogger } from '@backstage/backend-common';
```

Comment:
I'm moving this helper to the `backend-plugin-api` package, but I would like to confirm if moving to `backend-default` makes more sense.

Reply:
That sounds more like a testUtils thing.

Reply:
Should we use the `mockServices.logger.mock()` instead? This question just came to my mind now.

Reply:
Yeah!